code/game/objects/items/devices/radio/headset.dm

```diff
@@ -223,8 +223,8 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	AddComponent(/datum/component/wearertargeting/earprotection, list(ITEM_SLOT_EARS))
 
 /obj/item/radio/headset/headset_cent
```

Comment:
Same here, don't rename headset used by Rabbits, make a new one.

Reply:
New headset made for them.